packages/xo-web/src/common/intl/messages.js

```diff
@@ -1132,9 +1132,12 @@ const messages = {
   vifMacAutoGenerate: 'Auto-generated if empty',
   vifAllowedIps: 'Allowed IPs',
   vifNoIps: 'No IPs',
+  vifDisabledNetwork: 'Network disabled',
   vifLockedNetwork: 'Network locked',
   vifLockedNetworkNoIps:
     'Network locked and no IPs are allowed for this interface',
+  vifNotLockedNetworkWithIps:
+    'Network not locked and some IPs are allowed for this interface',
```

Comment:
This message is weird since you wouldn't understand why this is a "problem". AFAIU, having some allowed IPs on an unlocked network isn't a problem, it's just not necessary. So the message should probably look like:

"Network not locked: some IPs are unnecessarily set as allowed for this interface"